src/main/java/com/gdsc/studyjamapi/controller/UserController.java

```diff
@@ -25,6 +26,13 @@ public ResponseEntity<UserResponse> createUser(
     return ResponseEntity.created(locationOfNewUser).build();
   }
 
+  public ResponseEntity<UserResponse> editUser(
```

Comment:
You should include the annotation `@Override` above the method signature.

Reply:
Okey, lemme add to thiss method.